docs/guide/interactions/views.rst

```diff
@@ -0,0 +1,467 @@
+.. currentmodule:: discord
+
+.. _guide_interactions_views:
+
+Views
+=======
+
+Views are the primary way to design message component based UIs.
```

Comment:
What is a view? What does a view represent? Someone who would read through this page may not understand the concept of a view, as it is not explained yet.

Reply:
Added some prose, and linked to ddocs